scripts/access_control/eth/access_control_eoa.py

```diff
@@ -0,0 +1,128 @@
+from brownie import accounts, interface
+from ape_safe import ApeSafe
+
+ownable_contracts = [
+    "0x67b66c99d3eb37fa76aa3ed1ff33e8e39f0b9c7a",  # bank
+    "0x97a49f8eec63c0dfeb9db4c791229477962dc692",  # bank config
+    "0x61858a3d3d8fDbC622a64a9fFB5b77Cc57beCB98",  # goblin config
+    "0xbebbff645d666445f39900f33201405e1cdaf130",  # IbETHRouter
+]
+goblins = [
+    "0x3c2bbb353b48d54b619db8ac6aa642627fb800e3",
+    "0xd902a3bedebad8bead116e8596497cf7d9f45da2",
+    "0xf134fdd0bbce951e963d5bc5b0ffe445c9b6c5c6",
+    "0xd6419fd982a7651a12a757ca7cd96b969d180330",
+    "0x54a2c35d689f4314fa70dd018ea0a84c74506925",
+    "0xa7120893283cc2aba8155d6b9887bf228a8a86d2",
+    "0xcfbd9eeac76798571ed96ed60ca34df35f29ea8d",
+    "0xbb4755673e9df77f1af82f448d2b09f241752c05",
+    "0x35952c82e146da5251f2f822d7b679f34ffa71d3",
+    "0x8c5cecc9abd8503d167e6a7f2862874b6193e6e4",
+    "0x69fe7813f804a11e2fd279eba5dc1ecf6d6bf73b",
+    "0x8fc4c0566606aa0c715989928c12ce254f8e1228",
+    "0x37ef9c13faa609d5eee21f84e4c6c7bf62e4002e",
+    "0x9d9c28f39696ce0ebc42ababd875977060e7afa1",
+    "0x5c767dbf81ec894b2d70f2aa9e45a54692d0d7eb",
+    "0x6d0eb60d814a21e2bed483c71879777c9217aa28",
+    "0x795d3655d0d7ecbf26dd33b1a7676017bb0ee611",
+    "0x6a279df44b5717e89b51645e287c734bd3086c1f",
+    "0x1001ec1b6fc2438e8be6ffa338d3380237c0399a",
+    "0xa8854bd26ee44ad3c78792d68564b96ad0a45245",
+    "0x41f07d87a28adec58dba1d063d540b86ccbb989f",
+    "0xb7bf6d2e6c4fa291d6073b51911bac17890e92ec",
+    "0x4668ff4d478c5459d6023c4a7efda853412fb999",
+    "0x14804802592c0f6e2fd03e78ec3efc9b56f1963d",
+    "0xdaa93955982d32451f90a1109ecec7fecb7ee4b3",
+    "0x4d4ad9628f0c16bbd91cab3a39a8f15f11134300",
+    "0x4ec23befb01b9903d58c4bea096d65927e9462cc",
+    "0x66e970f2602367f8ae46ccee79f6139737eaff1c",
+    "0xcbb95b7708b1b543ecb82b2d58db1711f88d265c",
+    "0xfbc0d22bf0ecc735a03fd08fc20b48109cb89543",
+    "0xf285e8adf8b871a32c305ab20594cbb251341535",
+    "0xe900e07ce6bcdd3c5696bfc67201e940e316c1f1",
+    "0xee781f10ce14a45f1d8c2487aeaf24d0366fb9fa",
+    "0x09b4608a0ca9ae8002465eb48cd2f916edf5bf63",
+    "0x9d00b5eeedeea5141e82b101e645352a2ea960ba",
+    "0x0ec3de9941479526bb3f530c23aaff84148d17a7",
+    "0x6cc2c08e413638ceb38e3db964a114f139fff81e",
+    "0x766614adcff1137f8fced7f0804d184ce659826a",
+    "0xcc11e2cf6755953eed483ba2b3c433647d0f18dc",
+    "0x18712bcb987785d6679134abc7cddee669ec35ca",
+    "0xee8f4e4b13c610bfa2c65d968ba1d5263d640ce6",
+    "0xbd95cfef698d4d582e66110475ec7e4e21120e4a",
+    "0x9eed7274ea4b614acc217e46727d377f7e6f9b24",
+    "0x4354e09bb45ee1edc5ad97b44fa3682ef7e6c77e",
+    "0xa4bc927300f174155b95d342488cb2431e7e864e",
+]
+strats = [
```

Comment:
strats that missing
- 0x737aad349312f36b43041737d648051a39f146e8
- 0xc43dc00e5e4cfc8c8092ec4a5d363170c6d14ce9
- 0xe992e83b268f63ee508c6d292a54dff91c1eb57f
- 0x942d9e12bc440fe9c374e67dfb0328fb1fbfcd3d
- 0x58cc5c8e863759ba2aaae2bcaee84ce22404b5e7
- 0x81796c4602b82054a727527cd16119807b8c7608
- 0x466c427cc426a88ae2a596ab48a085dd72258354
- 0x14f66f8c283d004f4195cd041746b6b5fa823e16